account-kit/wallet-client/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@account-kit/wallet-client",
+  "version": "4.47.0",
+  "description": "Wallet Client for Alchemy Account Kit",
+  "author": "Alchemy",
+  "license": "MIT",
+  "private": false,
+  "type": "module",
+  "main": "./dist/esm/exports/index.js",
+  "module": "./dist/esm/exports/index.js",
+  "types": "./dist/types/exports/index.d.ts",
+  "typings": "./dist/types/exports/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist",
+    "src/**/*.ts",
+    "package.json",
+    "!dist/**/*.tsbuildinfo",
+    "!vitest.config.ts",
+    "!.env",
+    "!src/**/*.test.ts",
+    "!src/**/*.test-d.ts",
+    "!src/__tests__/**/*"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/types/exports/index.d.ts",
+      "require": "./dist/esm/exports/index.js",
+      "import": "./dist/esm/exports/index.js"
+    },
+    "./internal": {
+      "types": "./dist/types/exports/internal.d.ts",
+      "require": "./dist/esm/exports/internal.js",
+      "import": "./dist/esm/exports/internal.js"
+    },
+    "./abi": {
+      "types": "./dist/types/abi/index.d.ts",
+      "require": "./dist/esm/abi/index.js",
+      "import": "./dist/esm/abi/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "yarn clean && yarn build:esm && yarn build:types",
+    "build:esm": "tsc --project tsconfig.build.json --outDir ./dist/esm",
+    "build:types": "tsc --project tsconfig.build.json --declarationDir ./dist/types --emitDeclarationOnly --declaration --declarationMap",
+    "clean": "rm -rf ./dist",
+    "test:e2e": "bun test ./src/**/*.e2e.test.*"
+  },
+  "dependencies": {
+    "@aa-sdk/core": "^4.47.0",
+    "@account-kit/infra": "^4.47.0",
+    "@account-kit/smart-contracts": "^4.47.0",
+    "@alchemy/wallet-api-types": "0.1.0-alpha.11",
+    "@sinclair/typebox": "^0.34.33",
+    "deep-equal": "^2.2.3",
+    "ox": "^0.6.12",
+    "viem": "2.29.2"
+  },
+  "devDependencies": {
+    "@types/deep-equal": "^1.0.4",
+    "@types/node": "^24.0.11",
+    "bun": "^1.2.18",
```

Comment:
i'm curious how bun is used now that this is in a monorepo that uses yarn elsewhere. is it just for running tests? should we swap it out to just use vitest like the rest of the repo, if the interfaces are mostly the same & it's a quick swap?

Reply:
It's just for running the tests. vitest is missing some of the assertions used here, like `toBeArrayOfSize`. I tried porting it over to vitest but it seemed faster to just add bun only as the testing framework.

Reply:
sounds good